qa/tasks/cephfs/filesystem.py

```diff
@@ -810,7 +824,10 @@ def set_dir_layout(self, mount, path, layout):
 
     def add_data_pool(self, name, create=True):
         if create:
-            self.mon_manager.raw_cluster_cmd('osd', 'pool', 'create', name, self.pgs_per_fs_pool.__str__())
+            self.mon_manager.raw_cluster_cmd('osd', 'pool', 'create', name,
+                                             '--pg_num_min', str(self.pg_num_min),
+                                             '--target_size_ratio',
```

Comment:
Let's skip setting the target_size_ratio since we don't really know what it should be for an added data pool

Reply:
Done. Thanks.